openscm/timeframes.py

```diff
@@ -0,0 +1,65 @@
+class TimeframeConverter:
+    """
+    Converts timeseries and their points between two timeframes (each defined by a time
+    of the first point and a period length).
```

Comment:
Do we need the number of time periods/end point too? If no, I'm not clear what this does.

Reply:
We don't necessarily need the end points. The number of points/time periods would be used from the list/array in the `convert_X` function.

Reply:
got it!